website/content/commands/partition.mdx

```diff
@@ -66,6 +66,13 @@ You can issue the following subcommands with the `consul partition` command.
 ### `create`
 
 The `create` subcommand sends a request to the server to create a new admin partition.
+This subcommand has the following characteristics:
+- **[Required ACLs](/docs/security/acl/acl-system)**: `operator:write`
```

Comment:
I decided to link to the ACL docs in case someone reading this doesn't have ACLs enabled and isn't super familiar with them. Seem like a good idea?